### DISCUSSION TITLE
Monitor a running process

### DISCUSSION THREAD

**Discussion by BorisKourt** [babashka/process#97] [Q&A] [answered]
Hello! 

I've converted from [Conch](https://github.com/clj-commons/conch) to Process recently and its been a very smooth transition. 

During the conversion something I really wanted was to tap into/log the output of the running process before it completes. Is this something that is possible?

**Comment by borkdude** [comment, marked as answer]
@BorisKourt Do you just want to display the output or do you want to manipulate the output?

**Reply by BorisKourt** [reply]
Display would be enough if I am understanding correctly.

**Reply by borkdude** [reply]
@BorisKourt In that case I recommend the config option `:inherit true` which displays both stdout and stderr when the process is running: `(process [cmds] {:inherit true})`

The `shell` function does this automatically. Note that the syntax of `shell` differs from `process`.

```
(shell "ls -la")
```

**Reply by BorisKourt** [reply]
Thanks, that is perfect.
